website/addons/dataverse/tests/test_model.py

```diff
@@ -52,8 +62,10 @@ def test_node_addon_get_or_create_finds(self):
         assert_false(created2)
         assert_equals(dvf1, dvf2)
 
+    @mock.patch('website.addons.dataverse.model.request.cookies.get')
```

Comment:
Instead of monkeypatching `request`, you should be able to set up a test request context:

```python
from werkzeug.datastructures import Headers

cookie_header = Headers(['Cookie', '{}={}'.format(settings.COOKIE_NAME, self.cookie))
with self.app.app.test_request_context(headers=[cookie_header]):
    # ...
```

Doing so will make the test less brittle.